TeachingRecordSystem/tests/TeachingRecordSystem.Core.Tests/Services/TrsDataSync/TrsDataSyncServiceTests.Person.cs

```diff
@@ -69,7 +69,7 @@ await fixture.DbFixture.WithDbContext(async dbContext =>
     public async Task Contact_DeletedRecord_DeletesPersonRecordFromDatabase()
     {
         // Arrange
-        var createPersonResult = await TestData.CreatePerson(b => b.WithSyncOverride(true));
+        var createPersonResult = await TestData.CreatePerson(p => p.WithSyncOverride(true));
```

Comment:
WithTrn ?

Reply:
This one doesn't need it; TRN or not the record should be synced here.